File: backend/Amsterfam/Amsterfam.Core/Entities/PaymentMethod.cs
namespace Amsterfam.Core.Entities;

public class PaymentMethod
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public required string Title { get; set; }
    public string? Icon { get; set; }
    public string? Link { get; set; }
    public string? Description { get; set; }

    public User User { get; set; } = null!;
}

File: frontend/src/app/core/models/payment-method.ts
export interface PaymentMethod {
  id: number;
  title: string;
  icon: string | null;
  link: string | null;
  description: string | null;
}

export interface UpsertPaymentMethodRequest {
  title: string;
  icon: string | null;
  link: string | null;
  description: string | null;
}
